app/models/orchestration_template.rb

```diff
@@ -153,7 +153,7 @@ def md5=(_md5)
   end
 
   def self.calc_md5(text)
-    Digest::MD5.hexdigest(text) if text
+    Rails.application.config.digest_class.hexdigest(text) if text
```

Comment:
Ok thanks.  If we picked a new digest as default though, say SHA256, would it be possible to use that instead of MD5?